optuna/visualization.py

```diff
@@ -16,6 +22,9 @@
     _available = False
 
 
+TARGET_STATUS = [TrialState.PRUNED, TrialState.COMPLETE, TrialState.RUNNING]
```

Comment:
I think,
- "STATE" is better than "STATUS" 
- intermediate values may have a different meaning with final objective value, so it seems better to exclude `PRUNED` and `RUNNING` states from this list

```suggestion
TARGET_STATE = [TrialState.COMPLETE]
```